Code/ItemRandomizer.cs

```diff
@@ -543,17 +521,21 @@ public class Location
 				[JsonProperty("location")]
 				public string LocationName { get; set; }
 				public int Offset { get; set; }
+				[JsonProperty("scene")]
+				public string SceneName { get; set; }
 				public string Flag { get; set; }
 			}
 		}
 
-		private struct APFileData
+		public class APFileData
```

Comment:
Why is this a class now?

Reply:
It's a class because I need to check if object is null